sbt-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/ScalaJSBundlerPlugin.scala

```diff
@@ -557,7 +566,10 @@ object ScalaJSBundlerPlugin extends AutoPlugin {
         addPackages(baseDir, installDir, useYarn.value, log, npmExtraArgs.value, yarnExtraArgs.value)(s"jsdom@$jsdomVersion")
       }
       installDir
-    }
+    },
+
+    // Default to deprecated requiresDOM to not break old build.
+    requireJsDomEnv := requiresDOM.value
```

Comment:
This should be in `perConfigSettings` (or even just in `testSettings` if this is only used for testing?)

Reply:
Make sense. Will try to do that. 

I think the current design may not be as clean as it should. Probably we can just provide our own `jsEnv' and let users switch to it if necessary.

@cquiroz I remember you mentioned before that scala.js bundler relies on some scala.js internal. Is there a chance we can provide a `ScalaJSBundlerDomJsEnv` similar to https://github.com/scala-js/scala-js-dom ?